app/javascript/components/message.js

```diff
@@ -0,0 +1,28 @@
+
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { fetchMessages } from "../redux/message/messageSlice";
+
+const Message = () => {
+    const dispatch = useDispatch();
+    const message = useSelector((state) => state.messages);
+
+    useEffect(() => {
+        dispatch(fetchMessages());
+    }, [dispatch]);
+
+    return (
+        <>
+        <div>
+            <h2>Refresh this page to see new message</h2>
+            <ul>
+                {message.map((message) => (
+                    <li key={message.id}>{message.message}</li>
+                ))}
+            </ul>
+        </div>
+        </>
```

Comment:
- [ ] Kindly update your readme based on the template from microverse.

- [ ] Kindly ensure your greeting is displayed on the page. Currently, you are only logging it on the console.

  ![ejuw](https://user-images.githubusercontent.com/81468675/200341007-d6654c3c-0a8f-4d57-8cd3-bd955afe8d30.jpg)

Reply:
@julie-ify thanks for the feedback I have made changes in line with what you requested for